api/server.go

```
@@ -397,7 +396,7 @@ func (s *Server) GetTransactionsByBlockID(ctx context.Context, in *access.GetTra
 	}
 
 	chain := header.ChainID.Chain()
-	systemTx, err := blueprints.SystemChunkTransaction(chain)
+	systemTx, _ := blueprints.SystemChunkTransaction(chain)
```

why are we ignoring this error?